ingestion/tests/unit/metadata/cli/resources/profiler_workflow.py

```diff
@@ -1,7 +1,7 @@
 """
 This file has been generated from dag_runner.j2
 """
-from openmetadata.workflows import workflow_factory
+from openmetadata_managed_apis.workflows import workflow_factory
```

Comment:
oh were these not executed? I guess they'd fail. Maybe we need to review the Makefile

Reply:
Yes, it should have failed, not sure why it didn't.. didn't take a closer look, fixed the imports though